charts/argo-cd/templates/argocd-application-controller/role.yaml

```diff
@@ -19,6 +19,7 @@ rules:
   - argoproj.io
   resources:
   - applications
+  - applicationsets
```

Comment:
Hi @OpenGuidou ,
Since we follow upstream and we don't see the permission in v2.12.4. 🤔 
Ref: https://github.com/argoproj/argo-cd/blob/v2.12.4/manifests/base/application-controller-roles/argocd-application-controller-role.yaml

Can you please make PR for upstream ? Once upstream releases as a new version, we also follow this change.

Reply:
Sure !